### PR TITLE
Update scale-tidb-cluster.md

### PR DESCRIPTION
**Pull request by SunRunAway**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/scale-tidb-cluster.md [hunk 1]
```diff
@@ -109,23 +109,10 @@ For a running cluster, AWS and Google Cloud do not allow in-place storage capaci
 
 ## Increase node size
 
-When a cluster is running, you cannot increase its node size. To make such changes, take either of the following methods:
+When a cluster is running, you cannot increase its node size currently. You can only increase the node size through backup and restore.
```
Comment (ran-huang) [inline tidb-cloud/scale-tidb-cluster.md:112]:
```suggestion
Cuurently, when a cluster is running, you cannot increase its node size. You can only increase the node size through backup and restore.
```

File: tidb-cloud/scale-tidb-cluster.md [hunk 2]
```diff
@@ -109,23 +109,10 @@ For a running cluster, AWS and Google Cloud do not allow in-place storage capaci
...
 
-- Method 1: Increase the node size through backup and restore
+You need to [create a latest backup of the cluster](/tidb-cloud/backup-and-restore.md#manual-backup), [delete the cluster](/tidb-cloud/delete-tidb-cluster.md), and then increase the node size when you [restore the deleted cluster](/tidb-cloud/backup-and-restore.md#restore-a-deleted-cluster). Before taking this method, make sure the following impacts are acceptable:
 
-    You need to [create a latest backup of the cluster](/tidb-cloud/backup-and-restore.md#manual-backup), [delete the cluster](/tidb-cloud/delete-tidb-cluster.md), and then increase the node size when you [restore the deleted cluster](/tidb-cloud/backup-and-restore.md#restore-a-deleted-cluster). Before taking this method, make sure the following impacts are acceptable:
+- To avoid any data loss during or after the backup, you need to stop the connection to the cluster through your SQL client before creating the backup.
+- After you stop the connection to the cluster, your applications running on this cluster cannot provide service normally until the restoring process is completed.
```
Comment (ran-huang) [inline tidb-cloud/scale-tidb-cluster.md:118]: no consecutive blank lines. 
```suggestion
```

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

**Review by ran-huang — APPROVED** [review]

**Review by azurezyq — APPROVED** [review]

**Pull request merged**